.github/workflows/nightly.yaml

```diff
@@ -31,7 +31,7 @@ jobs:
       with:
         version: v0.13.0
     - uses: imjasonh/setup-crane@e82f1b9a8007d399333baba4d75915558e9fb6a4
-    - uses: sigstore/cosign-installer@v2.5.0
+    - uses: sigstore/cosign-installer@v3
```

Comment:
can you use the latest release(v3.0.1)?

Reply:
v3 is a mutable tag. Basically best practice for GitHub actions is to update the v3 tag to the latest release in v3.x which today is v3.0.1. We use that pattern in most places like with setup-go and checkout where we also specify only the major version and pick the latest patch level automatically.